astropy/io/fits/column.py

```diff
@@ -1095,7 +1095,7 @@ def _convert_to_valid_data_type(self, array):
             elif 'L' in format:
                 # boolean needs to be scaled back to storage values ('T', 'F')
                 if array.dtype == np.dtype('bool'):
-                    return np.where(array == False, ord('F'), ord('T'))
+                    return np.where(array == np.bool(False), ord('F'), ord('T'))
```

Comment:
This was also E712 (``E712 comparison to False should be 'if cond is False:' or 'if not cond:'``), and was one of the examples where the original must be used to be able to do element-wise comparison. So to get rid of the error notification I could either add a pylint disable comment, or use this workaround.